### PR TITLE
Jgfouca/branch for acme split 2018 02 22

### PR DESCRIPTION
**Pull request by jgfouca**
Test suite: 
Test baseline: 
Test namelist changes: 
Test status: [bit for bit, roundoff, climate changing]

Fixes [CIME Github issue #]

User interface changes?: 

Update gh-pages html (Y/N)?:

Code review:

### CODE REVIEW

File: scripts/lib/e3sm_cime_mgmt.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def get_branch_from_tag(tag):
 def do_subtree_split(old_split_tag, new_split_tag, merge_tag):
 ###############################################################################
     subtree_branch = get_branch_from_tag(new_split_tag)
-    run_cmd_no_fail("git subtree split {}.. --prefix=cime --onto={} -b {}".\
+    run_cmd_no_fail("git subtree split {}.. --prefix=cime --squash --onto={} -b {}".\
```
Comment (jgfouca) [inline scripts/lib/e3sm_cime_mgmt.py]: The squash and the explicit commit list are going to be removed

**Comment by jgfouca** [comment]
@billsacks , this is the first split with your recommended changes from https://github.com/ESMCI/cime/issues/2177#issuecomment-366413324 ( IE , no explicit commit list and no squash for the git subtree split ).

**Comment by billsacks** [comment]
@jgfouca I spot-checked some files and didn't notice any history problems. It was a bit hard to tell for sure because many of the changed files here already had history issues from earlier ACME merges (so there were a lot of lines in git blame that already started with `^`), but from what I can tell from my spot-checks, this iteration hasn't introduced any problems.

Thank you for making this change!

**Comment by billsacks** [comment]
So, just to be explicit about it: I'm comfortable with your merging this to master without `--squash` this time.

**Pull request merged**